### ISSUE TITLE
[Accessibility] - Screen reader announces invisible "indeterminate progress indicator" for quick pick list

### ISSUE THREAD

**Issue by tonybaloney** [microsoft/vscode#168271]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.73.1
- OS Version: macOS Ventura 13.0.1

Steps to Reproduce:

1. Navigate to an action in an extension that uses a Quick Pick List dialog box, for example "Select Display Language", Or the command palette
2. Use tab to try and navigate to the items in the quick pick list
3. Screen reader announces "indeterminate progress indicator" at the bottom of the quick input box for an invisible element.

https://user-images.githubusercontent.com/1532417/206094275-80a89019-53b0-47b5-85c5-462f1000a0fd.mp4

**Comment by tonybaloney** [comment]
Screen capture has system audio

**Comment by tonybaloney** [comment]
It is also very difficult to navigate to the items inside the quick pick list. The controls for this are not announced to the user.

**Comment by tonybaloney** [comment]
Element is a hidden progress indicator in every quick pick list:

```html
<div class="monaco-progress-container quick-input-progress done" role="progressbar" aria-valuemin="0" style=""><div class="progress-bit" style="background-color: rgb(56, 142, 60); width: inherit; opacity: 1;"></div></div>
```

**Comment by dmitrivMS** [comment]
Testing:
1) Following the repro steps/video above, navigate using NVDA (https://www.nvaccess.org/download/) in any access (Ctrl+P) or command (Ctrl+Shift+P) picker with items, make sure NVDA reads the item details and you don't hear anything about progress bar.
2) Make sure any access checkbox does not shift up and down as it is opened or when typing and filtering (i.e. the progress bar is hidden, but still occupies space in the dom). See https://github.com/microsoft/vscode/issues/272985 for repro of this issue.
3) If possible, try to make the progress bar appear (needs 3+ seconds of loading) - could use a code change or slow machine, make sure it still shows progress, then disappears.
